libs/errorHandler.js

```diff
@@ -3,18 +3,21 @@
 /*  Load gulp + plugins
 /* ------------------------------------------------------------------------- */
 
-var notify  = require("gulp-notify");
+var gutil = require('gulp-util');
```

Comment:
Basically, there's not much more you can output .. I will change this file later a bit more, using gulp-plumber, but for now it will do ..
Thanks for the merge :)